dvl-a50/dvl.py

```diff
@@ -62,6 +63,7 @@ class DvlDriver(threading.Thread):
     last_temperature_check_time = 0
     temperature_check_interval_s = 30
     temperature_too_hot = 45
+    confidence_cutoff = 0
```

Comment:
I'm a bit confused as to why this is 0 if the default is 80 - I think it could make more sense to specify a meaningful value here and then use that as the fallback when loading settings.